### PR TITLE
[Event Hubs] Processor Release Prep

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to prepare the Event Hubs Processor package for its May release.  This will be a stable (GA) release.

### CODE REVIEW

File: eng/Packages.Data.props [hunk 1]
```diff
@@ -138,7 +138,7 @@
     <PackageReference Update="Microsoft.AspNetCore.Http" Version="2.1.22" />
     <PackageReference Update="Microsoft.AspNetCore.Http.Connections" Version="1.0.15" />
     <PackageReference Update="Microsoft.Azure.Functions.Extensions" Version="1.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.1.1" />
+    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.1.2" />
```
Comment (jsquire) [inline eng/Packages.Data.props:141]: Not needed for Event Hubs, but riding along from today's Extensions package release.

**Review by christothes — APPROVED** [review]

File: eng/Packages.Data.props (hunk 1 above)
Comment (JoshLove-msft) [inline eng/Packages.Data.props:141]:
```suggestion
    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.2.0" />
```

Comment (jsquire) [inline eng/Packages.Data.props:141]:
```suggestion
    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.2.0" />
```

**Pull request merged**